plugins/lookup/nb_lookup.py

```diff
@@ -283,8 +290,8 @@ class LookupModule(LookupBase):
 
     def run(self, terms, variables=None, **kwargs):
 
-        netbox_api_token = kwargs.get("token")
-        netbox_api_endpoint = kwargs.get("api_endpoint")
+        netbox_api_token = kwargs.get("token") or os.getenv("NETBOX_TOKEN")
+        netbox_api_endpoint = kwargs.get("api_endpoint") or os.getenv("NETBOX_API") or os.getenv("NETBOX_TOKEN")
```

Comment:
I assume the last or should be `NETBOX_URL`, but should do that since the inventory plugin only uses the `NETBOX_API` environment. Do you mind adding a second option for token as well to match the inventory plugin? `NETBOX_API_KEY`.

Reply:
Makes perfect sense. My bad for missing it!